### DISCUSSION TITLE
ZT with AMSET + Alamode

### DISCUSSION THREAD

**Discussion by LauraCaputo** [ttadano/alamode#144] [Q&A]
Dear all,

I would like to calculate the ZT figure of merit of some materials. I have seen that AMSET (https://github.com/hackingmaterials/amset/tree/main) calculates S, sigma and Ke. If I understand correctly, it should calculate also tau.

So, can the outputs of AMSET be directly added to the Kp calculated by Alamode?

**Comment by ttadano** [comment]
This discussion is off-topic here. ALAMODE gives you lattice thermal conductivity, but how you combine this data with other quantities is entirely up to you.
